bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/ArchiveWorkItem.java

```diff
@@ -145,10 +145,18 @@ static String filterOutCommands(String body) {
 
     private boolean ignoreComment(HostUser author, String body, ZonedDateTime createdTime, ZonedDateTime lastDraftTime, boolean isComment) {
         if (pr.repository().forge().currentUser().equals(author)) {
-            return !PullRequestConstants.READY_FOR_SPONSOR_MARKER_PATTERN.matcher(body).find();
+            if (pr.isOpen()) {
+                return !PullRequestConstants.READY_FOR_SPONSOR_MARKER_PATTERN.matcher(body).find();
+            } else {
+                return true;
+            }
         }
         if (bot.ignoredUsers().contains(author.username())) {
-            return !PullRequestConstants.READY_FOR_SPONSOR_MARKER_PATTERN.matcher(body).find();
+            if (pr.isOpen()) {
+                return !PullRequestConstants.READY_FOR_SPONSOR_MARKER_PATTERN.matcher(body).find() && pr.isOpen();
```

Comment:
You already know `pr.IsOpen()` is true here.

Reply:
Oh right, thanks